.github/workflows/ci.yml

```diff
@@ -33,12 +33,18 @@ jobs:
         if: failure()
         run: mvn --batch-mode surefire-report:report-only
       - name: Publish unit test results
-        uses: docker://ghcr.io/enricomi/publish-unit-test-result-action:v1
+        if: always() && ( github.event_name == 'push' || !github.event.repository.fork )
```

Comment:
I think that it depends on the use case:
* When we want to identify a PR originating from a forked repo (i.e. to avoid duplicate runs), we should use `github.event.pull_request.head.repo.id != github.event.pull_request.base.repo.id`
* When we want to identify a push made in forked repo (i.e. to execute actions pertaining to a release), we should use `github.event.repository.fork`

This would ensure the right behaviour for a PR done on a forked repo from another forked repo, and also that release actions are only ever executed for the original repo.

Reply:
Exactly 👍